pandas/core/generic.py

```diff
@@ -7393,7 +7405,7 @@ def describe(self, percentiles=None, include=None, exclude=None):
         Excluding object columns from a ``DataFrame`` description.
 
         >>> df.describe(exclude=[np.object])
-                categorical  numeric
+               categorical  numeric
```

Comment:
When running the validation script, I occasionally get a failure

```
Line 210, in pandas.DataFrame.describe
Failed example:
    df.describe(exclude=[np.number])
Expected:
           categorical object
    count            3      3
    unique           3      3
    top              f      c
    freq             1      1
Got:
           categorical object
    count            3      3
    unique           3      3
    top              f      a
    freq             1      1
```

Did you see this at all? This likely is an issue in the method itself, and not the docstring.

Reply:
yeah i do see this error but its flaky.

Reply:
To be clear, it's probably some kind of non-stable sorting inside the describe method, and nothing wrong with the docstring. It may be best to just include the docstring, and open a new issue.

Reply:
The strange thing is that just doing

```python
pd.DataFrame({"A": pd.Categorical(['d', 'e', 'f']), "B": ['a', 'b', 'c'], 'C': [1, 2, 3]}).describe(exclude=['number'])
```

seems deterministic.